src/popup/bookmarkModule.js

```diff
@@ -16,7 +16,6 @@ export default function bookmarkImage(e) {
     if (bookmarksArray.indexOf(e.target.dataset.imageid) === -1) {
       bookmarksArray.push(e.target.dataset.imageid);
       chrome.storage.sync.set({ bookmarks: bookmarksArray }, () => {
-        e.target.innerText = 'bookmark';
```

Comment:
This statement was used to confirm and provide feedback to the user that the bookmark has been registered by changing the bookmark icon from [bordered](https://material.io/resources/icons/?icon=bookmark_border&style=baseline) to [filled](https://material.io/resources/icons/?icon=bookmark&style=baseline).
Instead of deleting, it should be modified to do the same thing with font awesome icons

Reply:
@makkoncept I changed it so that it displayed the filled bookmark icon when the image has been bookmarked.

It was also doing a thing where if you left the search module and then went back to it with the same search query, the icon was not displaying the filled in icon even though it had been previously bookmarked. I changed that in [this commit](https://github.com/creativecommons/ccsearch-browser-extension/pull/35/commits/5ce0f60aa7f6d7c3c6cd1f2f0bba6b5353fc44cc). It looks like a big change but the only things that changed were I [pulled the bookmarks from storage](https://github.com/creativecommons/ccsearch-browser-extension/pull/35/commits/5ce0f60aa7f6d7c3c6cd1f2f0bba6b5353fc44cc#diff-ac6db3f8e57ca4502560c69f517a9030R104-R105) and then [checked if that image has been bookmarked or not](https://github.com/creativecommons/ccsearch-browser-extension/pull/35/commits/5ce0f60aa7f6d7c3c6cd1f2f0bba6b5353fc44cc#diff-ac6db3f8e57ca4502560c69f517a9030R179-R191) and conditionally render outlined or filled.

Let me know what you think!